amato/umbrel-app.yml

```diff
@@ -0,0 +1,21 @@
+manifestVersion: 1
+id: amato
+category: productivity
+name: Amato Timer
+version: "0.0.1.1"
+tagline: Simple work timer.
```

Comment:
We can remove the `.` from the end of the tagline for style-consistency with the rest of the apps in the app store.